### ISSUE TITLE
[TW-1900] parsing `due` where we shouldn't

### ISSUE THREAD

**Issue by taskwarrior** [GothenburgBitFactory/taskwarrior#1913]
_Yanick Champoux on 2017-03-11T00:18:39Z says:_ 

 {code}
$ task add project:due-b Do Laundry
Cannot subtract strings
{code}

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2017-03-11T00:18:39Z
Modified: 2017-03-11T13:05:39Z
```

**Comment by taskwarrior** [comment]
_Yanick Champoux on 2017-03-11T00:19:26Z says:_ 

 Was found in https://github.com/yanick/Taskwarrior-Kusarigama/issues/14

**Comment by tbabej** [comment]
Fixed on 2.6.0:

```
[warrior@tw-2.6.0 ~]$ task add project:due-b Do Laundry
Created task 1.
The project 'due-b' has changed.  Project 'due-b' is 0% complete (1 task remaining).
[warrior@tw-2.6.0 ~]$ task all

ID St UUID     Age  Project Description
 1 P  68091e65 3s   due-b   Do Laundry

1 task
[warrior@tw-2.6.0 ~]$ 
```
